code/game/objects/items/weapons/defib.dm

```diff
@@ -398,7 +398,7 @@
 						for(var/obj/item/organ/external/O in H.organs)
 							total_brute	+= O.brute_dam
 							total_burn	+= O.burn_dam
-						if(total_burn <= 180 && total_brute <= 180 && !H.suiciding && !ghost && tplus < tlimit && !(NOCLONE in H.mutations) && (H.get_int_organ(/obj/item/organ/internal/heart) || H.get_int_organ(/obj/item/organ/internal/brain/slime)))
+						if(total_burn <= 180 && total_brute <= 180 && !H.suiciding && !H.hellbound && !ghost && tplus < tlimit && !(NOCLONE in H.mutations) && (H.get_int_organ(/obj/item/organ/internal/heart) || H.get_int_organ(/obj/item/organ/internal/brain/slime)))
```

Comment:
Well, the main issue is that the hellbound var has been added to all possible revive options like cloners and defibs, and that's a far from ideal solution. If a new way of reviving crew is ever added, that means the hellbound var needs to be inserted there as well which is pretty snowflakey.

I'd rather find a different solution to this problem - perhaps a more globalized variable/flag to prevent reviving that isn't directly tied to devils, or something along those lines.